### PR TITLE
Review code for Milestone 3

### PR DESCRIPTION
**Pull request by Datagirlcmr**
Features
Styled the app components using CSS

### CODE REVIEW

File: src/components/ButtonPanel.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import Button from './Button';
+
+function ButtonPanel() {
+  return (
+    <div className="cont-button">
+      <div className="btn-row">
+        <Button name="Ac" />
```
Comment (MauricioRobayo) [inline src/components/ButtonPanel.js]: Let's pass some color to the buttons that are not going to default to orange, for example:

```jsx
<Button name="Ac" color='#e0e0e0' />
```

**Review by MauricioRobayo — CHANGES_REQUESTED** [review]
Well done!

It looks great and it's coming into place, but 😥 we have to make sure all requirements are met before continuing.

- [ ] `App`, `Display` and `ButtonPanel` components use flex display mode   ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/useFlex.png)). Could you please use `display: flex` on the `App` and `ButtonPanel` components.
- [ ] You are not passing any color to the buttons on ButtonPanel. Your `Button` component takes the color property and it defaults to orange. Now, for the buttons that are not going to be orange, let's pass some color.

Status: __Request changes__.

File: src/components/ButtonPanel.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+import Button from './Button';
+
+function ButtonPanel() {
+  return (
+    <div className="cont-button">
+      <div className="btn-row">
+        <Button name="Ac" color="rgb(224, 223, 219)" />
+        <Button name="+/-" color="rgb(224, 223, 219)" />
+        <Button name="%" color="rgb(224, 223, 219)" />
+        <Button name="÷" color="rgb(224, 223, 219)" />
```
Comment (bellom) [inline src/components/ButtonPanel.js]: Good job making the necessary corrections seems you did more 😄 
You don't need to add color to the signs on the right side(signs like: `÷`, `x`,  `-`, `+`, `=`), we want them to have default color which is `orange` right.
kindly remove the color 👍 

Example: `<Button name="÷" />`


After I removed `rgb` color on side signs;
image

File: src/components/ButtonPanel.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+      </div>
+      <div className="btn-row">
+        <Button name="7" color="rgb(224, 223, 219)" />
+        <Button name="8" color="rgb(224, 223, 219)" />
+        <Button name="9" color="rgb(224, 223, 219)" />
+        <Button name="x" color="rgb(224, 223, 219)" />
```
Comment (bellom) [inline src/components/ButtonPanel.js]: kindly remove the `rgb` color 👍

File: src/components/ButtonPanel.js [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+      </div>
+      <div className="btn-row">
+        <Button name="4" color="rgb(224, 223, 219)" />
+        <Button name="5" color="rgb(224, 223, 219)" />
+        <Button name="6" color="rgb(224, 223, 219)" />
+        <Button name="-" color="rgb(224, 223, 219)" />
```
Comment (bellom) [inline src/components/ButtonPanel.js]: kindly remove the `rgb` color 👍

File: src/components/ButtonPanel.js [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+      </div>
+      <div className="btn-row">
+        <Button name="1" color="rgb(224, 223, 219)" />
+        <Button name="2" color="rgb(224, 223, 219)" />
+        <Button name="3" color="rgb(224, 223, 219)" />
+        <Button name="+" color="rgb(224, 223, 219)" />
```
Comment (bellom) [inline src/components/ButtonPanel.js]: kindly remove the `rgb` color 👍

File: src/components/ButtonPanel.js [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+      </div>
+      <div className="btn-row">
+        <Button name="0" wide color="rgb(224, 223, 219)" />
+        <Button name="." color="rgb(224, 223, 219)" />
+        <Button name="=" color="rgb(224, 223, 219)" />
```
Comment (bellom) [inline src/components/ButtonPanel.js]: kindly remove the `rgb` color 👍

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @Datagirlcmr   :clap: 

**Great work making `almost` all the necessary corrections from the previous review.**

**Kindly check out my comments below;**

**Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....**

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Review by shloch — APPROVED** [review]
accomplished
### Good to go ......

**Pull request merged**